comictaggerlib/ctsettings/commandline.py

```diff
@@ -160,14 +160,20 @@ def register_runtime(parser: settngs.Manager) -> None:
     parser.add_setting(
         "--json", "-j", action="store_true", help="Output json on stdout. Ignored in interactive mode.", file=False
     )
-
     parser.add_setting(
-        "-t",
-        "--type",
+        "--type-modify",
+        metavar=f"{{{','.join(metadata_styles).upper()}}}",
+        default=[],
+        type=metadata_type,
+        help="""Specify the type of tags to write.\nUse commas for multiple types.\nSee --list-plugins for the available types.\n\n""",
+        file=False,
+    )
+    parser.add_setting(
+        "--type-read",
         metavar=f"{{{','.join(metadata_styles).upper()}}}",
         default=[],
         type=metadata_type,
-        help="""Specify TYPE as either CR, CBL or COMET\n(as either ComicRack, ComicBookLover,\nor CoMet style tags, respectively).\nUse commas for multiple types.\nFor searching the metadata will use the first listed:\neg '-t cbl,cr' with no CBL tags, CR will be used if they exist\n\n""",
+        help="""Specify the type of tags to read.\nUse commas for multiple types.\nSee --list-plugins for the available types.\nThe tag use will be 'overlayed' in order:\ne.g. '-t cbl,cr' with no CBL tags, CR will be used if they exist and CR will overwrite any shared CBL tags.\n\n""",
```

Comment:
`-t` should be a short option for `--type-read` and `--type-modify` should should default to `--type-read` (implement it in `validate_commandline_settings`)